sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/configuration/AadResourceServerConfiguration.java

```diff
@@ -103,7 +103,7 @@ public List<OAuth2TokenValidator<Jwt>> createDefaultValidator(AadAuthenticationP
      * Default security configuration of the resource server, user can write another configuration bean to override it.
      */
     @EnableWebSecurity
-    @EnableGlobalMethodSecurity(prePostEnabled = true)
+    @EnableMethodSecurity
```

Comment:
Why making this change?

Reply:
The annotation @EnableGlobalMethodSecurity is marked as a deprecated API since the RC1 version, see https://github.com/spring-projects/spring-security/blob/6.0.0-RC1/config/src/main/java/org/springframework/security/config/annotation/method/configuration/EnableGlobalMethodSecurity.java